dair_pll/inertia.py

```diff
@@ -237,22 +390,41 @@ def drake_to_pi(spatial_inertia: DrakeSpatialInertia) -> Tensor:
             number_to_float(I_BBcm_B[index[0], index[1]])
             for index in INERTIA_INDICES
         ]
-        pi = Tensor([mass] + mass_list + inertia_list)
-        return pi
+        pi_cm = Tensor([mass] + mass_list + inertia_list)
+        return pi_cm
+
+    @staticmethod
+    def drake_to_pi_o(spatial_inertia: DrakeSpatialInertia) -> Tensor:
+        """Extracts a ``pi`` parameterization from a Drake
+        :py:attr:`~pydrake.multibody.tree.SpatialInertia` object."""
+        mass = number_to_float(spatial_inertia.get_mass())
+        p_BoBcm_B = spatial_inertia.get_com()
+        I_BBo_B = spatial_inertia.CalcRotationalInertia()
+
+        mass_list = [
+            mass * number_to_float(coordinate) for coordinate in p_BoBcm_B
+        ]
+
+        inertia_list = [
```

Comment:
Could this be made more compact using your new inertia vector <--> matrix conversions?

Reply:
It seemed like this could almost work.  One can do:
```
I_BBo_B_mat = I_BBo_B.CopyToFullMatrix3()
```
That creates a numpy array of the inertia matrix, however of `pydrake.symbolic.Expression` objects with `dtype=object`.  It's possible there's a way to convert an entire numpy array full of `pydrake.symbolic.Expression`s from `dtype=object` to `dtype=np.float64`, however I haven't found one yet.  Thus any solution that gets the inertia matrix via `RotationalInertia.CopyToFullMatrix()` (or other options like `RotationalInertia.get_moments()` and `RotationalInertia.get_products()`) would still require an element-by-element conversion, similar to `drake_to_pi_cm`'s current implementation.

Reply:
That's frustrating! seems like there isn't a good opportunity for improvement here then.